src/android/CameraLauncher.java

```diff
@@ -994,6 +994,7 @@ private Bitmap getScaledAndRotatedBitmap(String imageUrl) throws IOException {
                         // read exifData of source
                         exifData = new ExifHelper();
                         exifData.createInFile(filePath);
+                        exifData.readExifData();
```

Comment:
I still don't really understand how reading some data into a property/class variable of `exifData` influences the further execution of code. Can you explain?